app/controllers/users_controller.rb

```diff
@@ -18,9 +18,10 @@ def create
     if user.save
       session[:user_id] = user.id
       redirect_to dashboard_path
+      FriendNotifierMailer.inform(user).deliver_now
     else
       flash[:error] = user.errors.full_messages.to_sentence
-      render :new
+      #render :new
```

Comment:
commented this out because otherwise you are not able to register? Probably something we can fix

Reply:
Interesting, I wonder why it wouldnt work. This would be a great section to look at for a refactor.